src/client/decorators/DocumentDecorator/DiagramDesigner/DocumentEditorDialog.html

```diff
@@ -3,7 +3,7 @@
         <div class="modal-content">
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal">×</button>
-                <h3>Edit Comment Documentation</h3>
+                <h3 id="dialog-title">Edit Comment Documentation</h3>
```

Comment:
In general I would avoid using id attributes, but in this case since it's a modal it's not very probably that multiple ones are displayed at the same time. (And four lines down there's already an id attribute.

Reply:
So... do you want me to change it or not? :)

Reply:
You can leave it as it’s consistent with the other element and changing it might require changing some styles rules.